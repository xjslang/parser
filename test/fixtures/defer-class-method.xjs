class TestClass {
  testMethod() {
    console.log('method start')
    defer {
      console.log('method defer')
    }
    console.log('method end')
  }
}

const instance = new TestClass()
instance.testMethod()
