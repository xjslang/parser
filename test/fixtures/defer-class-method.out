method start
method end
method defer
